solomon/client.c

```diff
@@ -1,77 +1,36 @@
-#include "server.h"
+#include "sockets.h"
 
 int main(int argc, char * const argv[]){
     if(argc != 3) {
-        printf(
-		ANSI_COLOR_YELLOW 
-		"Usage: %s IP_ADDRESS PORT\n" 
-		ANSI_COLOR_RESET, 
-		argv[0]);
+        printf(ANSI_COLOR_YELLOW "Usage: %s IP_ADDRESS PORT\n" ANSI_COLOR_RESET, argv[0]);        
```

Comment:
Try to reach a conclusion alongside @FelSiq as to which syntax guideline you are going to take.
#1

Reply:
I was quite confused by this form of syntax guide that you used, it was complicated to know what code was only one line or was several, in some parts it might even help, but everything gets complicated to understand the code.

Let's make a poll to decide this.